pkg/measure-config/src/measurefs.go

```diff
@@ -21,9 +21,10 @@ import (
 
 const (
 	//TpmDevicePath is the TPM device file path
-	TpmDevicePath   = "/dev/tpmrm0"
-	configPCRIndex  = 14
-	configPCRHandle = tpmutil.Handle(tpm2.PCRFirst + configPCRIndex)
+	TpmDevicePath    = "/dev/tpmrm0"
+	configPCRIndex   = 14
+	configPCRHandle  = tpmutil.Handle(tpm2.PCRFirst + configPCRIndex)
+	maxEventDataSize = 1024
```

Comment:
How about adding comment that this is max size for PCREvent argument?

Reply:
done.